plugins/chat/spec/system/mention_warnings_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+RSpec.describe "Mentions warnings", type: :system, js: true do
+  fab!(:user) { Fabricate(:user) }
```

Comment:
one thing I love to do is calling this user: `current_user`, makes very clear this is the one being signed in